apps/web-giddh/src/app/invoice/generate/invoice.generate.component.html

```diff
@@ -79,7 +79,7 @@
                                 </div>
                                 <span>
                                     <button
-                                        *ngIf="!isCompany"
+                                        *ngIf="!isCompany || !isConsolidatedBranch"
                                         [hidden]="selectedLedgerItems?.length === 0"
```

Comment:
Check this condition in case of company mode branch consolidated mode and normal branch mode